packages/accounts/src/kernel-zerodev/types.ts

```diff
@@ -1,7 +1,11 @@
 import type { UserOperationCallData } from "@alchemy/aa-core";
+import type { Hex } from "viem";
 
-export interface KernelUserOperationCallData extends UserOperationCallData {
+export type KernelUserOperationCallData = Exclude<
+  UserOperationCallData,
+  Hex
+> & {
   delegateCall?: boolean;
-}
+};
```

Comment:
oh so does Exclude only work when `  UserOperationCallData` is the `Hex` type alias, vs. the calldata object of target, data, value?

Reply:
Other way around, the use of `Exclude` here means take all the union types of `UserOperationCallData` and omit the `Hex` one. Essentially: `Exclude<a | b, b> = a`